spec/features/super_admin/invite_user_to_organisation_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'support/invite_use_case'
+require 'support/notifications_service'
+
+describe "Inviting a team member as a super admin", type: :feature do
```

Comment:
I think there are some missing tests here, most can be copied from the admin invitation tests:
I think some of them will also be `request` tests instead of `feature` tests too.

```
without super admin privs, it ignores the provided parameters
with invalid email, it has the same behaviour as invalid email handling in admin invitations
```

What's the behaviour when tampering with parameters, (unset, set to non-existant org).
What's the behaviour of a re-invitation? Can a super-admin send a re-invitation?
If not, assert that it can't.

There's lots of undefined behaviour here.

Reply:
I've expanded on the tests, however in the future we should look at using a shared example to test common usecases when branching is involved